packages/core/src/animation/BlendTree.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity } from "./../Entity";
+import { Motion } from "./Motion";
+import { AnimationClip } from "./AnimationClip";
+
+export enum BlendTreeType {
+  Simple2D
+}
+
+/**
+ * TODO
+ * Blend trees are used to blend continuously animation between their childs. They can either be 1D or 2D.
+ */
+export class BlendTree extends Motion {
```

Comment:
You  can later, If it is not implemented yet, it can only be deleted

Reply:
done.